src/scripts/1-create-requests.ts

```diff
@@ -0,0 +1,167 @@
+import { IOracle } from '@defi-wonderland/prophet-sdk/dist/types/typechain';
+
+import { ContractRunner } from 'ethers-v6';
+import { getNetworkName } from '../utils';
+
+/**
+ * Create Requests script
+ *
+ * The goal of this script is to create a request using the sdk
+ *
+ * Requirements:
+ *  1. Deploy the Prophet modules you want to use to your preferred fork
+ *  2. Update the addresses of the oracle and the modules in `constants.ts`
+ *  3. Fund the requester wallet with the token you want to use, in this case we use USDT
+ *  4. Approve the modules running the `approve-modules.ts` script (only needed once)
+ */
+(async () => {
+  const signer = await getSigner();
+  console.log(`current signer: ${signer.address}`);
+  console.log(`current network: ${getNetworkName()}`);
+
+  const runner = signer as unknown as ContractRunner;
+
+  // The modules are defined to be used in the sdk as known modules
+  const httpRequestModule = {
+    module: new Module(address.deployed[getNetworkName()].HTTP_REQUEST_MODULE, IHttpRequestModule.abi, runner),
+    abi: IHttpRequestModule.abi,
+  };
+
+  const bondedResponseModule = new Module(
+    address.deployed[getNetworkName()].BONDED_RESPONSE_MODULE,
+    IBondedResponseModule.abi,
+    runner
+  );
+
+  const bondedDisputeModule = {
+    module: new Module(address.deployed[getNetworkName()].BONDED_DISPUTE_MODULE, IBondedDisputeModule.abi, runner),
+    abi: IBondedDisputeModule.abi,
+  };
+
+  const arbitratorModule = {
+    module: new Module(address.deployed[getNetworkName()].ARBITRATOR_MODULE, IArbitratorModule.abi, runner),
+    abi: IArbitratorModule.abi,
+  };
+
+  const callbackModule = {
+    module: new Module(address.deployed[getNetworkName()].CALLBACK_MODULE, ICallbackModule.abi, runner),
+    abi: ICallbackModule.abi,
+  };
+
+  const knownModules: ModulesMap = {
+    [httpRequestModule.module.moduleAddress]: httpRequestModule.module,
+    [address.deployed[getNetworkName()].BONDED_RESPONSE_MODULE]: bondedResponseModule,
+    [bondedDisputeModule.module.moduleAddress]: bondedDisputeModule.module,
+    [arbitratorModule.module.moduleAddress]: arbitratorModule.module,
+    [callbackModule.module.moduleAddress]: callbackModule.module,
+  };
+
+  // The sdk is initialized with the runner, the oracle address and the known modules
+  const sdk = new ProphetSDK(runner, address.deployed[getNetworkName()].ORACLE, knownModules);
+
+  // usdt is going to be used as the bond token
+  const { usdt } = await getERC20Tokens();
+
+  const accountingExtension = await getAccountingExtension();
+
+  const deadline = FINALIZE_SOON ? getCurrentEpochInSeconds() + 30 : getRandomDeadline();
+
+  // Define the data to be sent for each module
+  const requestModuleData = Object.values({
+    url: 'https://defi.sucks/',
+    body: 'test body',
+    method: 0,
+    accountingExtension: address.deployed[getNetworkName()].ACCOUNTING_EXTENSION,
+    paymentToken: await usdt.getAddress(),
```

Comment:
Could you store accounting extension address and USDT address in a variable? I believe it will make the code a bit cleaner.